modules/models/src/main/java/org/eclipse/xpanse/modules/models/workflow/migrate/enums/MigrationStatus.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.models.workflow.migrate.enums;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.xpanse.modules.models.common.exceptions.UnsupportedEnumValueException;
+
+/**
+ * Defines possible states of a Migration Service.
+ */
+public enum MigrationStatus {
+
+    MIGRATION_STARTED("migration_started"),
```

Comment:
all string values should be without `underscore`. See `ServiceState` for example.